Signal/test/util/OWSDatabaseConverterTest.m

```diff
@@ -150,6 +141,8 @@ - (void)openYapDatabase:(NSString *)databaseFilePath
             [expectation fulfill];
         });
 
+        // YapDatabase can retain references to the registration
+        // connections for up to 5 seconds.
```

Comment:
I don't understand this comment. It sounds like it's related to https://github.com/signalapp/Signal-iOS/pull/3214

My understanding is that we're waiting 5 seconds in this test simply because this is an async process that will take "a while", and 5 secs was a safe upper bound.

Ultimately, I'm not super concerned about nitpicking a comment in a test, but I want to make sure I understand whats going on.

Reply:
Elaborated.